google-cloud-bigquery/lib/google/cloud/bigquery/dataset.rb

```diff
@@ -1014,7 +1023,7 @@ def load table_id, file, format: nil, create: nil, write: nil,
                       delimiter: delimiter, ignore_unknown: ignore_unknown,
                       max_bad_records: max_bad_records, quote: quote,
                       skip_leading: skip_leading, dryrun: dryrun,
-                      schema: schema }
+                      schema: schema_gapi }
```

Comment:
I've played around with this and this shouldn't be a problem. It looks like `schema_gapi` is `nil` and not undefined.